.github/workflows/prebuild.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  push:
+    tags:
+      - 'v*'
+
+name: Prebuild
+
+jobs:
+  prebuild-node:
+    name: Upload Node Prebuild
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npm install
```

Comment:
Why are no tests run here, even though they are run for the Electron prebuilds?

Reply:
we need to run tests here as well.